File: demo-spring-boot/demo-spring-boot-mybatis/src/main/java/com/future/demo/bean/Student.java
package com.future.demo.bean;

import java.io.Serializable;
import java.math.BigDecimal;
import lombok.Data;

/**
 * 
 * @TableName student
 */
@Data
public class Student implements Serializable {
    /**
     * 
     */
    private Long id;

    /**
     * 
     */
    private String name;

    /**
     * 
     */
    private Integer age;

    /**
     * 
     */
    private BigDecimal score;

    private static final long serialVersionUID = 1L;
}

File: demo-c++/demo-c-plus-class/student.cpp
#include "student.h"

// Student 构造函数调用基类 Person(name, age) 构造函数
Student::Student(std::string name, int age, int grade)
    :Person(name, age)
{
    this->grade = grade;
    // 调用基类人类打招呼方法
    this->sayHi();
}

Student::~Student() {

}
